election-common/Cargo.toml

```diff
@@ -6,5 +6,10 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+concordium-rust-sdk = { path = "../deps/concordium-rust-smart-contracts/concordium-rust-sdk/", optional = true }
 rmp-serde = "1.1"
-serde = "1.0"
+serde = {version = "1.0", features = ["derive"]}
+
+[features]
+default = ["full"]
+full = ["dep:concordium-rust-sdk"]
```

Comment:
I would prefer here to just depend on contracts_common without features.

But maybe it's fine for this.

Reply:
Yeah lets do that instead, good suggestion 🙂